### PR TITLE
Additional output plugin for writing to OpenTSDB using telnet mode

### PR DESCRIPTION
**Pull request by rplessl**
Hi,

This additonal output plugin enables to write from telegraf to OpenTSDB using the telnet mode.

My Unit Tests are not 100% code coverage, at the moment I will write a MOC for OpenTSDB emulation which will follows in a second pull request.

I have signed already the CLA

Thanks for pulling in advance!

### CODE REVIEW

File: outputs/opentsdb/opentsdb.go [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+import (
+	"fmt"
+	"net"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/influxdb/influxdb/client"
+    "github.com/influxdb/telegraf/outputs"
```
Comment (sparrc) [inline outputs/opentsdb/opentsdb.go]: please run `go fmt` on your code, it looks like you have some space-indented lines here

Comment (rplessl) [inline outputs/opentsdb/opentsdb.go]: fixed thanks for the hint

**Comment by sparrc** [comment]
This is great, thanks much @rplessl! I can tell that you read the contributing guidelines :-) :+1: :+1: 

re: unit tests, I'm not sure how much work it's going to be to mock openTSDB, but have you considered trying out a docker image of openTSDB rather than mocking it? The kafka output sink utilizes that for unit testing and it simplified things enormously.

**Comment by rplessl** [comment]
thanks for the hint for unit tests- I will also take a docker image

**Comment by sparrc** [comment]
You can do one or the other....basically would require adding the docker image to the scipts/docker-compose.yml file, then using testutil to get the host that the docker openTSDB is at. That would also make it part of a "long" unit test suite

**Comment by rplessl** [comment]
Hi @sparrc: 
As discussed last week, I have added several unit tests using OpenTSDB a docker image.  
Could you pull it?

**Comment by sparrc** [comment]
thanks @rplessl, I'm going to try to merge this today :+1:

**Pull request closed without merging**